src/specs/matrix4.spec.js

```diff
@@ -149,6 +149,138 @@ describe('Matrix4', () => {
     });
   });
 
+  describe('prototype.lookAt()', () => {
+    test('creates view matrix with positive offset from eye to center, and z up vector', () => {
+      const eye = Vector3.of(1.0, 1.0, 1.0);
+      const center = Vector3.of(10.0, 10.0, 10.0);
+      const up = Vector3.of(0.0, 0.0, 1.0);
+
+      const matrix = new Matrix4().lookAt(eye, center, up);
+
+      /* eslint-disable */
+      expect(matrix.approximates(Matrix4.of(
+         0.7071068287, -0.4082481861,  0.5773502588, 0.0, // col 0
+        -0.7071067095, -0.4082484245,  0.5773502588, 0.0, // col 1
+         0.0,           0.8164966106,  0.5773502588, 0.0, // col 2
+         0.0,           0.0,          -1.732050776,  1.0  // col 3
+      ))).toBe(true);
+      /* eslint-enable */
```

Comment:
What does eslint complain about here? Alignment?

Reply:
Yep. The left side doesn't match the expected indentation.